components/Document/DocumentViewer.tsx

```diff
@@ -534,10 +534,11 @@ const styles = StyleSheet.create({
     width: "100%",
     height: "100%",
     marginTop: 0,
-    overflow: "scroll",
+    overflow: "auto",
```

Comment:
Can you check commenting works okay in viewer? Also can you check different resolutions just to make sure.

Reply:
yes it's fine -- we should always have `overflow: "auto"` over `overflow: "scroll"` -- `overflow: "scroll"` will show the scrollbars always, but auto only shows the scrollbars when you scroll